### PR TITLE
apx: add page

### PR DESCRIPTION
**Pull request by kbdharun**
Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** `apx --version 1.1.8`

### CODE REVIEW

File: pages/linux/apx.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# apx
+
+> Package management utility for Vanilla OS.
+> It installs packages inside managed containers or directly inside the host.
```
Comment (kbdharun) [inline pages/linux/apx.md:4]:
```suggestion
> Install packages inside managed containers or directly inside the host.
```
Would `Install` packages be better here?

File: pages/linux/apx.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Install a package in the system and also initialize the container:
+
+`sudo apx install --sys {{package}} && apx init`
+
+- Install package(s) in system:
+
+`sudo apx install --sys {{package1 package2 ...}}`
+
+- Install a package from AUR (Use `apx --aur run {{package}}` , `apx --aur remove {{package}}` to run or remove an installed package):
```
Comment (kbdharun) [inline pages/linux/apx.md:15]: IMO, we can leave the extra commands here instead of creating a separate page. What do you guys think?

Comment (blueskyson) [reply pages/linux/apx.md:15]: I suggest use something like `sudo apx {{--sys|--aur}} install {{package}}` to include sys and aur in one example.

**Review by blueskyson — APPROVED** [review]

File: pages/linux/apx.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+> Install packages inside managed containers or directly inside the host.
+> More information: <https://github.com/Vanilla-OS/apx>.
+
+- Install a package in the system and initialize the container:
+
+`sudo apx install --sys {{package}} && apx init`
+
+- Install package(s) in the system or install AUR package(s) inside a container:
+
+`sudo apx install {{--sys|--aur}} {{package1 package2 ...}}`
```
Comment (navarroaxel) [inline pages/linux/apx.md:13]: We use this format
```suggestion
`sudo apx install --{{sys|aur}} {{package1 package2 ...}}`
```

File: pages/linux/apx.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`sudo apx --sys update`
+
+- Upgrade all installed packages in the system to their newest available version:
+
+`sudo apx --sys upgrade`
+
+- Remove package(s) in the system or from the AUR container:
+
+`sudo apx {{--sys|--aur}} remove {{package1 package2 ...}}`
```
Comment (kbdharun) [inline pages/linux/apx.md:29]:
```suggestion
`sudo apx --{{sys|aur}} remove {{package1 package2 ...}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**